example/package-lock.json

```diff
@@ -10252,9 +9897,9 @@
           }
         },
         "react": {
-          "version": "16.13.1",
-          "resolved": "https://registry.npmjs.org/react/-/react-16.13.1.tgz",
-          "integrity": "sha512-YMZQQq32xHLX0bz5Mnibv1/LHb3Sqzngu7xstSM+vrkE5Kzr9xE0yMByK5kMoTK30YVJE61WfbxIFFvfeDKT1w==",
+          "version": "16.14.0",
```

Comment:
We define the dependency in package.json like this:
"react": "^16.13.1",
It allowed react to bump to 16.14.0